app/routes/application.js

```diff
@@ -2,10 +2,11 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   actions: {
-    save: function() {
+    save() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).